gk/main.c

```diff
@@ -566,6 +566,21 @@ gk_flow_tbl_bucket_scan(uint32_t *bidx,
 				instance->ip_flow_hash_table, fe);
 		}
 
+		if (fe->state == GK_REQUEST) {
```

Comment:
We can use the body of this `if` statement as the `case GK_REQUEST:` in `is_flow_expired()`. Not only does this change improve the readability of the code here, but it also (1) eliminates the parameter `request_timeout_cycles` of `is_flow_expired()`, (2) eliminates the configuration parameter `gk_conf->request_timeout_cycles`, and avoids repeating the code here in `find_flow_entry_candidate()`. This change is better suited in its own patch.